test/uploads.js

```
@@ -1,41 +1,38 @@
             }
           }
-          // eslint-disable-next-line handle-callback-err
-        }, function (err, payload) {})
-      }).then(function (activityId) {
-        // clean up the uploaded activity
-        strava.activities.delete({ id: activityId }, function (err, payload) {
-          if (err) console.log(err)
-          done()
         })
-      })
+      });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)